### PR TITLE
Timer format

### PR DESCRIPTION
**Pull request by EstatoDeviato**

### CODE REVIEW

File: server/commands/roleplay.py [hunk 1]
```diff
@@ -944,3 +944,40 @@ def ooc_cmd_trigger(client, arg):
...
+            areas = [client.area]
+        else:
+            client.send_ooc("You cannot change timer format if you are at least CM")
+            return
+    timer.format = args[1]
+    if timer.set:
+        current_time = int(timer.static.total_seconds()) * 1000
+        for area in areas:
+            for c in area.clients:
+                c.send_command("TF", args[0], args[1], current_time)
```
Comment (Crystalwarrior) [inline server/commands/roleplay.py:982]: Maybe re-use send_timer_set_time instead?

**Review by Crystalwarrior — CHANGES_REQUESTED** [review]
Looks good, only 1 change requested to the /format_timer command before merge

**Comment by Crystalwarrior** [comment]
Oh yeah-- can you also add a documentation/commands.md entry and any aliases to the config_sample?

**Comment by EstatoDeviato** [comment]
Okay I've added stuff to command aliases and to documentation

File: server/client_manager.py [hunk 2]
```diff
@@ -300,6 +300,11 @@ def send_timer_set_time(self, timer_id=None, new_time=None, start=False):
                 else:
                     self.send_command("TI", timer_id, 2, new_time) # Show timer
                     self.send_command("TI", timer_id, int(not start), new_time) # Set timer with value and start
+                    if timer_id == 0:
+                        timer = self.area.area_manager.timer
+                    else:
+                        timer = self.area.timers[timer_id-1]
+                    self.send_command("TF", timer_id, timer.format, new_time)
```
Comment (Crystalwarrior) [inline server/client_manager.py:307]: does DRO support timer formatting or is this just a KFO-Client feature we came up with? I'll double-check.

**Review by Crystalwarrior — APPROVED** [review]
Cool and Good, just testing left

**Comment by Crystalwarrior** [comment]
Closed by #145

**Pull request closed without merging**